serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc

```diff
@@ -24,90 +22,93 @@ kubernetes:<group>/<version>/<kind>/<namespace>/<resourceName>?<attributeName>=<
                                                                    - labels=label-name=label-value;other-label=other-value
 ----
 
-The following scheme values are supported:
+Therefore, you must keep in mind that if application startup time matters, then consider to use a static URL instead. 
```

Comment:
isn't this sentence lost here?

Reply:
Done, changed the paragraph above the URI pattern as follows:

The Kubernetes service discovery feature works during the workflow application startup, in which this feature scans all the Quarkus configuration in search of the URI pattern. Therefore, you must keep in mind that if application startup time matters, then consider to use a static URL instead.

Following is the URI pattern in Kubernetes service discovery: